code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -944,24 +944,16 @@
 	damage = config.super_hit_damage
 	max_range = config.norm_shell_range
 
-/datum/ammo/rocket/wp/drop_flame(turf/T)
-	if(!istype(T))
+/datum/ammo/rocket/wp/drop_flame(radius = 3, turf/T) //~Art updated fire.
+	if(!T || !isturf(T))
 		return
 	smoke.set_up(1, T)
 	smoke.start()
-	if(locate(/obj/flamer_fire) in T)
-		return
-	new /obj/flamer_fire(T, pick(15, 20, 25, 30))
-	for(var/mob/living/carbon/M in range(3, T))
-		if(isXeno(M))
-			var/mob/living/carbon/Xenomorph/X = M
-			if(X.fire_immune)
-				continue
-		if(M.stat == DEAD)
-			continue
-		M.adjust_fire_stacks(rand(5, 25))
-		M.IgniteMob()
-		M.visible_message("<span class='danger'>[M] bursts into flames!</span>","[isXeno(M)?"<span class='xenodanger'>":"<span class='highdanger'>"]You burst into flames!</span>")
+	for(var/obj/flamer_fire/F in range(radius,T)) // No stacking flames!
+		cdel(F)
+	playsound(T, 'sound/weapons/gun_flamethrower2.ogg', 50, 1, 4)
+	flame_radius(radius, 35, 35, 35, 35)
```

Comment:
```suggestion
	flame_radius(radius, 24, 17, 24, rand(5,20)) //regular flamer fire stats
```
Flamer fire `burn_intensity` (`burnlevel`) is 24.
Flamer fire `burn_duration` (`burntime`)  = 17
`burn_damage` is not really needed as a var, as it overlaps with `burn_intensity`
`fire_stacks` is capped at 20, and the flamer fire uses `rand(5,burnlevel*2)`
If anything, this should be milder.